homeassistant/components/ezviz/update.py

```diff
@@ -0,0 +1,105 @@
+"""Support for EZVIZ sensors."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyezviz import HTTPError, PyEzvizError
+
+from homeassistant.components.update import (
+    UpdateDeviceClass,
+    UpdateEntity,
+    UpdateEntityDescription,
+    UpdateEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DATA_COORDINATOR, DOMAIN
+from .coordinator import EzvizDataUpdateCoordinator
+from .entity import EzvizEntity
+
+PARALLEL_UPDATES = 1
+
+UPDATE_ENTITY_TYPES: dict[str, UpdateEntityDescription] = {
+    "version": UpdateEntityDescription(
+        key="version",
+        name="Firmware Update",
```

Comment:
The entity class sets the new naming entity style (`has_entity_name`), yet this name doesn't comply with that.

```suggestion
        name="Firmware update",
```

Reply:
Fixed. Apologies for this one.